src/app/clusters/general-diagnostics-server/general-diagnostics-server.cpp

```diff
@@ -15,11 +15,13 @@
  *    limitations under the License.
  */
 
+#include "app/server/Server.h"
 #include <app-common/zap-generated/attributes/Accessors.h>
 #include <app-common/zap-generated/cluster-objects.h>
 #include <app-common/zap-generated/ids/Attributes.h>
 #include <app-common/zap-generated/ids/Clusters.h>
 #include <app/AttributeAccessInterface.h>
+#include <app/CommandHandlerInterface.h>
```

Comment:
Why is this being included?  It's not needed....

Reply:
Weird, I could have sworn I needed it for `CommandHandler`. But turns out I don't

Reply:
You need `CommandHandler.h` for that.